frontend/src/features/Auth/components/LogoutModal/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { Button, Modal, ModalBody, ModalFooter, ModalHeader, Flex, Image } from "@yamada-ui/react";
+import { useRouter } from "next/router";
+import { useCallback } from 'react';
+
+const LogoutModal = () => {
+
+    const router = useRouter()
+    const handleCancel = useCallback(() => {
+        router.back()
+    }, [router])
+
+    const logout = useCallback(() => {
+        // 1.ログアウト処理(cookieで実装)
+
+        // 2.signInに遷移
+        router.push("/signin")
+    }, [])
+
+    return (
+      <div>
+        <Modal isOpen={true}>
+        <Image
+            src="https://not-found.com/not-found.png"
+            fallback="https://via.placeholder.com/512"
+            size="px"
+        />
+        <ModalHeader display={"flex"} justifyContent={"center"}>Xからログアウトしますか</ModalHeader>
+            <ModalBody>
+            いつでもログインし直すことができます。アカウントを切り替える場合は、既存のアカウントを追加すると切り替えることができます。
+            </ModalBody>
+
+            <ModalFooter justifyContent={"center"}>
+                <Flex direction={"column"}>
+                    <Button variant="ghost" onClick={logout}>
+                    ログアウト
+                    </Button>
+                    <Button colorScheme="primary" onClick={handleCancel}>キャンセル</Button>
+                </Flex>
```

Comment:
デザイン的にはこっちのほうが良さそうかなと思った

<img width="396" alt="スクリーンショット 2024-06-27 14 29 44" src="https://github.com/givery-bootcamp/dena-2024-team4/assets/84154073/2bc20df1-4e76-44b5-86f4-9075597ba608">


```suggestion
                <Flex direction="column">
                  <Button colorScheme="primary" onClick={logout} marginBottom="4px">
                    ログアウト
                  </Button>
                  <Button variant="outline" onClick={handleCancel}>
                    キャンセル
                  </Button>
                </Flex>
```

Reply:
参考になるコードを教えてくれてありがとう！
本家に寄せて修正しました！

<img width="472" alt="スクリーンショット 2024-06-28 11 31 13" src="https://github.com/givery-bootcamp/dena-2024-team4/assets/56498440/76c30256-ef8f-4c71-854e-03c88d640b6b">

<img width="401" alt="image" src="https://github.com/givery-bootcamp/dena-2024-team4/assets/56498440/427d553c-a060-4c7a-9486-226b4d751b8a">